README.md

```diff
@@ -202,6 +202,91 @@ Note that as of November 6, 2023, OpenShift Serverless Operator is based on RHEL
 
     > **Note:** that every minute on the clock a job is triggered to reconcile the CRs with the chart values. These cronjobs are deleted when their respective features (e.g. `rhdhOperator.enabled=false`) are removed or when the chart is removed. This is required because the CRs are not managed by helm due to the CRD dependency pre availability to the deployment of the CR.
 
+### Using Knative kafka broker
+If you want to use a Knative broker for communication between the different componenets (Data Index, Job Service and Workflows), you should use a reliable broker, ie: not in-memory.
+
+Kafka perfectly fullfil this reliability need.
+
+For that, you will need:
+
+1. A Kafka cluster running, see https://strimzi.io/quickstarts/ for a quickstart setup
+1. To configured and enable kafka broker feature in Knative:
+```console
+oc apply --filename https://github.com/knative-extensions/eventing-kafka-broker/releases/download/knative-v1.14.5/eventing-kafka-controller.yaml
```

Comment:
Should we include also a recommendation for the version mentioned in this context? Which openshift-serverless operator supports `v1.14.5`? is it the same versioning?

Reply:
I changed that a bit to give the official installation guide and I only provide this command as example, I put a warning about the version